### DISCUSSION TITLE
Set and view payment currency on Order edit page.

### DISCUSSION THREAD

**Discussion by samuelbirch** [craftcms/commerce#2381] [Ideas]
Commerce 3 cp orders.

If there is multiple payment currencies setup can we have an option in cp orders to choose which currency please.

Thanks

**Comment by lukeholder** [comment]
Yeah, we need to add payment currency either to the order edit page or the CP payment modal.

**Comment by samuelbirch** [comment]
my preference would be the edit order page, so the admin can see the final price before paying for it. 

Most of our customers use cp orders to take phone orders and for the sites that have multiple currencies it would be ideal to change it on the page where they are seeing the total price.

**Comment by lukeholder** [comment]
- [ ] Let the store admin change the payment currency before completing the order
- [ ] Show the payment currency being used on the payment modal
- [ ] Show the payment currency used in the meta sidebar
